Conceptpower+Spring/src/main/java/edu/asu/conceptpower/core/ConceptEntry.java

```diff
@@ -29,43 +29,43 @@
     private static final long serialVersionUID = 4569090620671054560L;
 
     @Id
-    @LuceneField(lucenefieldName = LuceneFieldNames.ID, isTokenized = false)
+    @LuceneField(lucenefieldName = LuceneFieldNames.ID, isTokenized = false, isCommaSeperated = false)
     private String id;
 
     @SearchField(fieldName = SearchFieldNames.WORDNETID)
-    @LuceneField(lucenefieldName = LuceneFieldNames.WORDNETID, isTokenized = false)
+    @LuceneField(lucenefieldName = LuceneFieldNames.WORDNETID, isTokenized = false, isCommaSeperated = true)
     private String wordnetId;
 
     @SearchField(fieldName = SearchFieldNames.WORD)
-    @LuceneField(lucenefieldName = LuceneFieldNames.WORD, isTokenized = true)
+    @LuceneField(lucenefieldName = LuceneFieldNames.WORD, isTokenized = true, isCommaSeperated = false)
```

Comment:
a better name for this would be ```isMultiple```. This will make more sense going forward in case we still need this when we update the data model to actually using lists rather than strings.